spec/services/descriptions_grouper_spec.rb

```diff
@@ -183,7 +682,7 @@
       expect(run.dig(druid2, "form3.type")).to eq("genre")
       expect(run.dig(druid2, "form4.value")).to eq("electronic resource")
       expect(run.dig(druid2, "form4.type")).to eq("form")
-      expect(run.dig(druid2, "form5.value")).to eq("1 CD-ROM ; 4 3/4 in. + 1 user's guide (loose-leaf ; 23 cm.)")
+      expect(run.dig(druid2, "form5.value")).to eq("1 CD-ROM ; 4 3/4 in. + 1 users guide (loose-leaf ; 23 cm.)")
```

Comment:
Can you say more about these quote changes in the expectations?

Reply:
Yes. I got rid of all of the single and double quotes from the test JSON. They are irrelevant to the test and made the JSON harder to deal with.

Reply:
OK, no problem.